tensorflow/contrib/verbs/rdma.cc

```diff
@@ -480,10 +800,10 @@ void RdmaChannel::Connect(const RdmaAddress& remoteAddr) {
     memset(&attr, 0, sizeof(ibv_qp_attr));
     attr.qp_state = IBV_QPS_RTR;
     struct ibv_port_attr port_attr;
-    CHECK(!ibv_query_port(adapter_->context_, (uint8_t)1, &port_attr))
```

Comment:
I think we can remove this query_port since this was done only for taking the active mtu, and we are not using it anymore.